ksqldb-parser/src/main/java/io/confluent/ksql/util/QueryGuid.java

```diff
@@ -21,27 +21,27 @@
 
 public final class QueryGuid {
   private final String clusterNamespace;
-  private final String queryUuid;
-  private final String anonQueryUuid;
+  private final String queryGuid;
+  private final String structuralGuid;
   private final LocalDateTime timeOfCreation;
 
   public QueryGuid(final String namespace, final String nonAnonQuery, final String anonQuery) {
     this.clusterNamespace = namespace;
-    this.queryUuid = computeQueryId(nonAnonQuery, clusterNamespace);
-    this.anonQueryUuid = computeQueryId(anonQuery, "");
+    this.queryGuid = computeQueryId(nonAnonQuery, clusterNamespace);
+    this.structuralGuid = computeQueryId(anonQuery, "");
```

Comment:
can we get rid of `getClusterNamespace`? it seems to be an internal detail and is only called from tests